ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -300,10 +339,15 @@ private static String getDefaultOrigin(String uri) {
     }
   }
 
+  @ReactMethod
+  public void setBinaryType(String binaryType, int id) {
+    mBlobsEnabled.put(id, binaryType.equals("blob"));
```

Comment:
Why are blobs not automatically enabled for any web socket if the data type is "binary"?